README.md

```diff
@@ -103,6 +104,14 @@ Environment Variables:
 
 Use VS Code feature [Multi-root Workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces). Each folder in workspace is treated as addon root.
 
+### How can I debug into third party library code from within my addon code?
+
+As described above an addon can be debugged when started from VS Code using the `Blender: Start` command.
+By default, debug breakpoints work only for files and directories opened in the current workspace and it is also not possible to step into code that is not part of the workspace.
+Nevertheless, if you explicitly need to debug third party Python module code, e.g., `site-packages` or Blender internal Python files, you can disable the [`blender.addon.justMyCode`](vscode://settings/blender.addon.justMyCode) extension setting in VS Code.
+You can do so by setting `"blender.addon.justMyCode": false,` in your user `settings.json` or you may also find this setting by searching for `just my code` under `File -> Preferences -> Settings`.
+After changing this setting, breakpoints and step into (<kbd>F11</kbd>) should work also for non-workspace files.
```

Comment:
wayy too long. And come on, I assume that you know how to change setting in vs code, right? Right?

```suggestion
Addon can be debugged when started from VS Code using the `Blender: Start` command. 
By default, debug breakpoints work only for files and directories opened in the current workspace and it is also not possible to step into code that is not part of the workspace.
Disable the VS Code setting [`blender.addon.justMyCode`](vscode://settings/blender.addon.justMyCode) to debug code anywhere.
In rare cases debugging with VS Code can crash Blender (ex. https://github.com/JacquesLucke/blender_vscode/issues/188).
```

>  it is also not possible to step into code that is not part of the workspace.

it that really the case?

Reply:
Yes it is the case, this is actually my main problem because I set breakpoints in my own code that I know and then from there I want to step into the third party functions.